react/src/base/inputs/Select.stories.js

```diff
@@ -158,9 +149,150 @@ export const disabledSelectBox = () => (
 disabledSelectBox.story = {
   name: 'Disabled',
   parameters: {
-    jest: [
-      'SprkSelectionInput',
-    ]
+    jest: ['SprkSelectionInput'],
   },
 };
 
+export const hugeSelectBox = () => (
+  <SprkSelectionInput
+    label="Select Box Label"
+    choices={[
+      {
+        label: 'Option 1',
+        value: 'option-1',
+      },
+      {
+        label: 'Option 2',
+        value: 'option-2',
+      },
+      {
+        label: 'Option 3',
+        value: 'option-3',
+      },
+      {
+        label: 'Grouped Options',
+        options: [
+          {
+            label: 'Grouped Option 1',
+            value: 'grouped-option-1',
+          },
+          {
+            label: 'Grouped Option 2',
+            value: 'grouped-option-2',
+          },
+          {
+            label: 'Grouped Option 3',
+            value: 'grouped-option-3',
+          },
+        ],
+      },
+    ]}
+    variant="hugeSelect"
```

Comment:
We added a huge select?

Reply:
Same reasoning here. We wanted to add examples because people didn't know it was available.